src/storage.ts

```diff
@@ -507,12 +508,12 @@ export class Storage {
 			: path.join(this.globalStorageUri.fsPath, "url");
 	}
 
-	public writeToCoderOutputChannel(message: string) {
-		this.output.appendLine(`[${new Date().toISOString()}] ${message}`);
-		// We don't want to focus on the output here, because the
-		// Coder server is designed to restart gracefully for users
-		// because of P2P connections, and we don't want to draw
-		// attention to it.
+	/**
+	 * Compatibility method for Logger interface used by headers.ts and error.ts.
+	 * Delegates to the logger module.
+	 */
+	public writeToCoderOutputChannel(message: string): void {
+		logger.info(message);
```

Comment:
Oh wait I thought you changed `headers.ts` and `error.ts`.  Can we delete this function now?